### PR TITLE
Clarivate extension test

### PR DESCRIPTION
**Pull request by darshan-clvt**

### CODE REVIEW

File: src/api/dao/dao-utils.js [hunk 1]
```diff
@@ -18,9 +18,9 @@ daoUtils.isUserAdmin = (userInfo) => {
             let groupId = userInfo.groups[i];
             for (let groupIndex = 0; groupIndex < groups.groups.length; ++groupIndex) {
                 const group = groups.groups[groupIndex];
-                if (groupId != group.id) {
-                    continue;
```
Comment (saichrn7) [inline src/api/dao/dao-utils.js:22]: @darshan-clvt , why you removed existing code ? any reason ?

File: src/ui/routes/login.js [hunk 2]
```diff
@@ -146,10 +146,11 @@ Error description: ${errorDescription}`, next);
                 debug(userInfo);
 
                 // Note that this is only to make sure the UI behaves; there are some specialties
-                // for admins and approvers. There are backend checks for all of these things as well
```
Comment (saichrn7) [inline src/ui/routes/login.js:149]: @darshan-clvt , keep backend here

File: src/ui/views/user.pug [hunk 3]
```diff
@@ -159,7 +159,15 @@ block content
 
             br
             br
-            if authUser.admin
+            p #{JSON.stringify(authUser)}
```
Comment (saichrn7) [inline src/ui/views/user.pug:162]: @darshan-clvt , remove this p tag..

File: src/ui/views/user.pug [hunk 4]
```diff
@@ -159,7 +159,15 @@ block content
...
+            if authUser.superadmin
```
Comment (saichrn7) [inline src/ui/views/user.pug:163]: @darshan-clvt , remove this if else block too

File: src/ui/views/user.pug [hunk 5]
```diff
@@ -168,19 +176,45 @@ block content
                         for group in groups
                             .checkbox
                                 label
-                                    if group.isMember
+                                    if group.isMember 
                                         input(type='checkbox' name=group.id value=group.id checked)
-                                        | #{group.name}
+                                        | #{group.name,"this is from if"}
```
Comment (saichrn7) [inline src/ui/views/user.pug:181]: @darshan-clvt , remove all debuuging related changes you have added

**Review by saichrn7 — CHANGES_REQUESTED** [review]
@darshan-clvt , I have added the some comments,work on those

File: src/api/dao/dao-utils.js (hunk 1 above)
Comment (darshan-clvt) [reply src/api/dao/dao-utils.js:22]: you told me comment it out!..

File: src/ui/routes/login.js (hunk 2 above)
Comment (darshan-clvt) [reply src/ui/routes/login.js:149]: okay done.

**Pull request closed without merging**